src/pages/change-request-details/change-request-details.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { AxiosResponse } from 'axios';
+import { ChangeRequest, exampleStandardChangeRequest } from 'utils';
+import { apiFetch } from '../../shared/axios';
+import ChangeRequestDetailsView from '../../components/change-request-details/change-request-details';
+import './change-request-details.module.css';
+
+const ChangeRequestDetails: React.FC = () => {
+  interface ParamTypes {
+    id: string;
+  }
+  const { id } = useParams<ParamTypes>();
+  const [changeRequest, setChangeRequest] = useState<ChangeRequest>(exampleStandardChangeRequest); // store projects data
+
+  // Transforms given project data and sets local state
+  const updateData: (response: AxiosResponse) => void = (res) => {
+    setChangeRequest({
+      ...res.data,
+      dateSubmitted: new Date(res.data.dateSubmitted),
+      dateReviewed: new Date(res.data.dateReviewed),
+      dateImplemented: new Date(res.data.dateImplemented)
+    });
+  };
+
+  // Fetch change request from API on component loading
+  useEffect(() => {
+    let mounted = true; // indicates component is mounted
+
+    const fetchChangeRequest: Function = async () => {
+      apiFetch
+        .get(`/change-requests/${id}`)
+        .then((response: AxiosResponse) => (mounted ? updateData(response) : ''))
+        .catch((error) =>
+          mounted ? console.log('fetch change request error: ' + error.message) : ''
+        );
+    };
+    fetchChangeRequest();
+
+    // cleanup function indicates component has been unmounted
+    return () => {
+      mounted = false;
+    };
```

Comment:
Very similar to fetch function in line 41 of change-request-table container. Can probably lift into helper function in separate file if will use again.

Reply:
Yes, but I'm too lazy and I'd prefer to do that as a separate ticket.

Reply:
Fair enough